### ISSUE TITLE
Debug logging of ARH is overriden

### ISSUE THREAD

**Issue by kiwixz** [lieser/dkim_verifier#422]
[Here](https://github.com/lieser/dkim_verifier/blob/183c81230835d9b2180c7d9be1492ea469bcfa95/modules/authVerifier.mjs.js#L348C44-L348C44) the result of ARH parsing is logged.  I believe it would be more convenient to log a deep clone or JSON.stringify of the object because it seems that it currently gets modified later.  The Thunderbird console, trying to be helpful, will actually show the updated object...  In my case its the result of the subsequent attempt to validate the DKIM which ends up being DKIM_SIGERROR_CORRUPT_BH.

**Comment by lieser** [comment]
I agree it is not helpful if the logged data changes afterwards. Maybe a few other places are also affected,

I will probably go with deep cloning. I kind of like the more structured logging.
The only issue is that then reporting bus people sometimes fail to expand the relevant information.

**Comment by lieser** [comment]
Will be fixed in the next version.
